supabase/migrations/20260826100543_harden_regenerate_hashed_apikey_rpc.sql

```
@@ -0,0 +1,175 @@
+  END IF;
+
+  IF NEW.principal_type = public.rbac_principal_apikey() THEN
+    PERFORM public.lock_rbac_apikey_principal(NEW.principal_id);
+  END IF;
+
+  RETURN NEW;
+END;
+$$;
+
+ALTER FUNCTION public.lock_rbac_apikey_principal_on_binding() OWNER TO postgres;
+REVOKE ALL ON FUNCTION public.lock_rbac_apikey_principal_on_binding() FROM PUBLIC;
```

SQLFluff (4.3.0)

[error] Line is too long (82 > 80).

(LT05)